02 ReactApp/src/pods/organization/component.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class OrganizationComponent extends Component {
+
+  componentDidMount() {
+    this.props.fetchMembers();
+  }
+
+  render() {    
+    return (
```

Comment:
Good points, I wanted to keep it with few lines of code, but now that we are thinking about already start with the component implemented, I would go with the second approach and I would separate into a different file (for just an li it could look like an overkill but for any tabular / row stuff is not a bad idea IMHO), now that we have this code like that should we switch to to a table like? (we can show the avatar an login name)